firestore-bigquery-export/scripts/import/src/config.ts

```diff
@@ -0,0 +1,188 @@
+import * as inquirer from "inquirer";
+import * as program from "commander";
+
+import { CliConfig } from "./types";
+
+const BIGQUERY_VALID_CHARACTERS = /^[a-zA-Z0-9_]+$/;
+const FIRESTORE_VALID_CHARACTERS = /^[^\/]+$/;
+
+const FIRESTORE_COLLECTION_NAME_MAX_CHARS = 6144;
+const BIGQUERY_RESOURCE_NAME_MAX_CHARS = 1024;
+
+const validateBatchSize = (value: string) => {
+  return parseInt(value, 10) > 0;
+};
+
+const validateLocation = (value: string) => {
+  const index = [
+    "us-west4",
```

Comment:
this list of location names seems a little brittle, since we'll have to quickly update and re-release every time a new location is added. Any chance there's an API for this?

Reply:
Hmm, not that I know of - I believe it would need to be updated similar to the main extension `yaml` files.

Is an Api something that we could consider in the future?

Reply:
Sounds good, thanks for checking. Guess we'll have to stick to manual updates for now!